lib/langchain/tool/calculator.rb

```diff
@@ -33,6 +33,7 @@ def execute(input:)
     rescue Eqn::ParseError, Eqn::NoVariableValueError
       # Sometimes the input is not a pure math expression, e.g: "12F in Celsius"
       # We can use the google answer box to evaluate this expression
+      # TODO: SERP_API_KEY is now being passed into SerpApi constructor, so I think below line won't work as is
       hash_results = Langchain::Tool::SerpApi.execute_search(input: input)
```

Comment:
@andreibondarev Everything seems to work except this line.  It's failing on the api_key and I don't think the Calculator tool should be calling this directly, but I'm unsure how to fix it.  Might need to pass in the search instance to Calculator#execute?

Reply:
Actually -- don't worry about it for now, I'll probably just get rid of this.

Reply:
OK.  I made the changes you noted so far, but there's 1 test marked 'xit' now because it will fail until this is fixed.